android/sdk/src/main/java/io/o2mc/sdk/Datastreams/Connectivity.java

```diff
@@ -15,58 +15,76 @@
  * Class pulled from Github. (With some small changes)
  * It returns the current networkstate as a String
  */
+@SuppressWarnings("WeakerAccess")
 public class Connectivity {
 
     /**
      * Get the network info
+     *
      * @param context
-     * @return
+     * @return null if unable to retrieve active network information
      */
-    public static NetworkInfo getNetworkInfo(Context context){
+    @SuppressWarnings("JavaDoc")
+    public static NetworkInfo getNetworkInfo(Context context) {
         ConnectivityManager cm = (ConnectivityManager) context.getSystemService(Context.CONNECTIVITY_SERVICE);
+        //noinspection ConstantConditions -- null is being dealt with in calling methods
         return cm.getActiveNetworkInfo();
     }
 
     /**
      * Check if there is any connectivity
+     *
      * @param context
      * @return
      */
-    public static boolean isConnected(Context context){
+    @SuppressWarnings("JavaDoc")
```

Comment:
not sure why we would want to suppress this javadoc warning

Reply:
same reason as with the previous 'context'